### ISSUE TITLE
USAGov Accessibility Review

### ISSUE THREAD

**Issue by r-bartlett-gsa** [GSA/usagov-benefits-eligibility#473]
[Accessibility test results provided by USAGov](https://drive.google.com/drive/folders/1DDRdPe9Wjc6JeN9JcSoIcU5SXLVfQPI_?usp=sharing)

- [x] Row 25: Unnecessary list coding is used for How to Apply and under Sign Up 
- [x] Row 33: Incorrect label for the Sorting field
- [x] Row 38: Grey status is not identified
- [x] Row 44: Insufficient color contrast for error message
- [x] Row 47: Unable to use arrow keys for radio buttons
- [x] Row 65: Link with the same text (agency name) but going to different location
- [x] Row 77: Missing submit button / issue with focus after selecting criteria
- [x] Row 84: Error message is not descriptive
- [x] Row 85: Formatting guidance is not included in date field label
- [x] Row 87: Error handling not present on page
- [x] Row 94: Apply button appears as button but is coded as link
- [x] Row 95: No indication on action occurred for Open All / Close All and for individual accordions
- [x] Row 96: No indication on action occurred for Clear Selections (and when removing benefit type filter)
- [x] Row 97: Confusing functionality with sort/filter in the accordion
- [x] Row 98: The tool top is not available for screen reader users
- [x] Open all and Close All look like links but are coded as buttons. 
- [ ] With the screen reader running, when the user activates the Skip to main content link the focus is not set  to H1 -  The anchor is set to the main content
- [ ] No indication on action occurred when accordions are expanded/collapsed

**Comment by fpigeonjr** [comment]
> With the screen reader running, when the user activates the Skip to main content link the focus is not set to H1 - The anchor is set to the main content

I am unable to duplicate this issue

**Comment by fpigeonjr** [comment]
> Row 97: Confusing functionality with sort/filter in the accordion

This may have some UI/UX implications. May need to regroup on this issue.

**Comment by fpigeonjr** [comment]
> Open all and Close All look like links but are coded as buttons.

This may have some UI/UX implications. May need to regroup on this issue.

**Comment by fpigeonjr** [comment]
> No indication on action occurred when accordions are expanded/collapsed

With screen-reader running, opening an accordion announces that it is "expanded" or "collapsed" as seen below:
<img width="1485" alt="Screen Shot 2022-06-17 at 3 24 41 PM" src="https://user-images.githubusercontent.com/4629398/174397032-fe991299-cbaa-40f4-b817-3d76ca1753ee.png">

**Comment by r-bartlett-gsa** [comment]
Feedback: "more info needed" should not display on the accordion heading by default. It should only be displayed if any selections on the left are made. See #489

**Comment by r-bartlett-gsa** [comment]
See retest results #499
